backend/app/adapter/db/changelog.go

```diff
@@ -40,7 +40,7 @@ FROM "%s";`,
 		if err != nil {
 			return changeLog, err
 		}
-		change.ReleasedAt = utc(change.ReleasedAt)
+		change.ReleasedAt = *utc(&change.ReleasedAt)
```

Comment:
```suggestion
		change.ReleasedAt = change.ReleasedAt.UTC()
```
The utc helper was created to conveniently handle `nil` value. `change.ReleasedAt` can't be `nil` here. So that's not a problem.